modules/yarpc/yarpc.go

```diff
@@ -230,17 +230,17 @@ func (c *dispatcherController) mergeConfigs(name string) (conf yarpc.Config, err
 
 // Creates a new YARPC module and adds a common middleware to the global config collection.
 // The first created module defines the service name.
-func newYARPCModule(
+func newModule(
 	mi service.ModuleCreateInfo,
 	reg registerServiceFunc,
 	options ...modules.Option,
-) (*YARPCModule, error) {
+) (*Module, error) {
 	name := "yarpc"
```

Comment:
I looked up current code and don't see a separate name being set for thrift. Maybe I'm missing something? Which file are you looking at Peter?